Shared/Samples/Get elevation at point on surface/GetElevationAtPointOnSurfaceView.swift

```diff
@@ -0,0 +1,131 @@
+    
+    /// The location callout placement.
+    @State private var calloutPlacement: CalloutPlacement?
+    
+    /// The screen point where to get the base surface location.
+    @State private var screenPoint: CGPoint!
+    
+    /// The scene point where the scene was tapped.
+    @State private var scenePoint: Point!
+    
+    /// The surface elevation of the tapped point.
+    @State private var elevation: Double?
+    
+    var body: some View {
+        SceneViewReader { sceneViewProxy in
+            SceneView(scene: model.scene)
+                .onSingleTapGesture { screenPoint, scenePoint in
+                    self.screenPoint = screenPoint
+                    self.scenePoint = scenePoint
+                }
+                .callout(placement: $calloutPlacement.animation(.default.speed(2))) { _ in
+                    VStack(alignment: .leading) {
+                        Text("Elevation")
+                            .font(.headline)
+                        Text(model.elevationString(elevation: elevation))
+                            .font(.callout)
+                    }
+                    .padding(5)
+                }
+                .task(id: scenePoint) {
+                    guard let scenePoint,
+                          // Converts the tapped screen point into a point on the surface.
+                          let relativeSurfacePoint = sceneViewProxy.baseSurfaceLocation(fromScreenPoint: screenPoint) else { return }
+                    if calloutPlacement == nil {
+                        // Gets the elevation from the tap location.
+                        elevation = await elevation(at: relativeSurfacePoint)
+                        // Shows the callout at the tapped location.
+                        calloutPlacement = CalloutPlacement.location(scenePoint)
+                    } else {
+                        // Hides the callout.
+                        calloutPlacement = nil
+                    }
+                }
+        }
+    }
+    
+    /// Gets the elevation at the surface point.
+    /// - Parameter surfacePoint: A geographical location on the base surface
+    /// in the same spatial reference as the scene view.
+    /// - Returns: The elevation in meters.
+    private func elevation(at surfacePoint: Point) async -> Double? {
+        return try? await model.scene.baseSurface.elevation(at: surfacePoint)
+    }
+}
+
+private extension GetElevationAtPointOnSurfaceView {
+    /// The model used to store the geo model and other expensive objects
+    /// used in this view.
+    class Model: ObservableObject {
+        /// A scene with imagery basemap.
+        let scene: ArcGIS.Scene = {
+            let scene = Scene(basemapStyle: .arcGISImagery)
+            
+            // Sets the initial viewpoint of the scene to northern Nepal.
+            scene.initialViewpoint = Viewpoint(
+                latitude: 28.42,
+                longitude: 83.9,
+                scale: 1e4,
```

Comment:
There is an ongoing discussion on a better way to create an `initialCamera` for a scene in Swift repo. These values can be arbitrary.

Reply:
How would you feel about leaving them .zero for now since they're ineffective? I think that'd help highlight the important parameters here.

Reply:
I'll change them to `.nan`